USAGE.md

```diff
@@ -8,9 +8,15 @@ Install rust: https://www.rust-lang.org/tools/install
 
 Install docker: https://docs.docker.com/desktop/
 
+### Setup environment variables
+
+Create a `signer.env` at the root of the file and add fields according to the [example file](./.env.example/signer.env.example). 
```

Comment:
You don't need the environment variable if you use docker as it is explained in this guide.
the docker-compose file, set all environment variables.

You can remove this.

Reply:
add the infos about signer.env in the readme of the crate you are working with. That's where it is used